projects/graph/social/social.py

```diff
@@ -44,11 +45,15 @@ def populateGraph(self, numUsers, avgFriendships):
         self.lastID = 0
         self.users = {}
         self.friendships = {}
-        # !!!! IMPLEMENT ME
-
         # Add users
-
+        for i in range(numUsers):
+            self.addUser(i)
         # Create friendships
+        for j in range(numUsers):
+            for k in range(random.randint(0, avgFriendships+1)):
+                r_int = random.randint(1, numUsers)
+                print("r_int", r_int)
+                self.addFriendship(j, r_int)
```

Comment:
The only suggestion I have here, and this applies overall, is don't commit one liners or spacing changes, that's not very helpful, also try to be a little more specific with your commits. 
I will refer to this guide:
https://github.com/erlang/otp/wiki/writing-good-commit-messages
If you wish to keep a brief title for example you can add comments in the body of the commit.